protoutil/testdata/peer-expired.pem

```diff
@@ -0,0 +1,14 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
it would be good to track the command used to generate this fixture

Reply:
I copied this cert from the msp testdata... do you know if the msp tests mention how they were generated?

Reply:
ha! no, unfortunately.. I think we would need to script the regeneration of those fixture (or at least document how we generated them). For another PR though